scripts/pre_commit_linter.py

```diff
@@ -29,11 +30,16 @@
 
     at least once.
 
-2.  This script should be run from the oppia root folder:
+=====================
+CUSTOMIZATION OPTIONS
+=====================
+1.  To lint only files that have been touched in this commit
+       python scripts/pre_commit_linter.py
 
-        python scripts/pre_commit_linter.py
+2.  To lint all files in  the folder or to lint just a specific file 
+        python scripts/pre_commit_linter.py dir_path
```

Comment:
Hm, I would have expected something like

    python scripts/pre_commit_linter.py --path filepath

(note the '--path' bit)